apps/web/pages/_document.tsx

```diff
@@ -77,7 +77,7 @@ class MyDocument extends Document<Props> {
         </Head>
 
         {/* Keep the embed hidden till parent initializes and gives it the appropriate styles */}
-        <body className="dark:bg-darkgray-50 bg-gray-100">
+        <body>
```

Comment:
New Shell requires background to be not gray

Reply:
I suppose this will affect v1 users, how can we prevent that while merging this to mail?